backend/lib/use-cases/case-notes/case-notes.ts

```diff
@@ -81,8 +81,8 @@ export class CaseNotesUseCase {
     const dateOfEdit = new Date().toISOString();
     const newNote: CaseNote = {
       ...existingNote,
+      ...note,
       id: randomUUID(),
-      documentType: 'NOTE',
```

Comment:
I know I said we don't need this, but it probably should stay. If the client were to change the documentType, the new "note" would be invalid.